controller/src/main/java/cern/jarrace/controller/Application.java

```diff
@@ -0,0 +1,25 @@
+package cern.jarrace.controller;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+
+/**
+ * Main entry point for the controller, instanciates an embedded Jetty container server, loads the spring
+ * {@link org.springframework.web.servlet.DispatcherServlet} and configures all the
+ * {@link org.springframework.web.bind.annotation.RestController}s
+ * @author timartin
+ */
+
+@SpringBootApplication
+public class Application {
+    public static void main(String args[]) {
```

Comment:
Usually I put the main in the main configuration file, because then I just separate all configuration files by responsabilities and include them in this main one, which has the only responsibility of initializing the whole thing